VsCredentialProvider/Sample.VisualStudio.CredentialProvider/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
```

Comment:
lets leave it with packages.config until we can do dotnetcore `packagereference`